CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_stripe_source v0.13.0
```

Comment:
Same note from the transform, let's update the changelog accordingly given the new default state of the new variable (false).

Reply:
updated